pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -110,11 +126,10 @@ func NewConfigObserver(
 			network.ObserveServiceClusterIPRanges,
 			nodeobserver.NewLatencyProfileObserver(
 				node.LatencyConfigs,
-				nodeobserver.NewSuppressConfigUpdateUntilSameProfileFunc(
-					operatorClient.(v1helpers.StaticPodOperatorClient),
-					kubeInformersForNamespaces.ConfigMapLister().ConfigMaps(operatorclient.TargetNamespace),
-					node.LatencyConfigs,
-				),
+				[]nodeobserver.ShouldSuppressConfigUpdatesFunc{
+					extremeProfileSuppressor,
```

Comment:
let's put a comment here that the first one should be the more important one as it will resolve first. @swghosh can you also please put such a comment to KASo?

Reply:
Ahh, for KASO it won't be required as KASO don't require `extremeProfileSuppressor` and it'll just have the `differentConfigProfileSuppressor`. Because for KubeAPIServer: Medium and Low has the same set of arg val pairs, thus no rejection is required there!

[1] https://coreos.slack.com/archives/C0392N52Q5P/p1653567422152599?thread_ts=1653567121.768759&cid=C0392N52Q5P
[2] https://github.com/openshift/enhancements/blob/master/enhancements/worker-latency-profile/worker-latency-profile.md#mediumupdateaveragereaction---lowupdateslowreaction

Reply:
Ok cool, thanks for the comment